package.json

```diff
@@ -76,7 +76,6 @@
     "@redhat-cloud-services/frontend-components-translations": "3.2.6",
     "@redhat-cloud-services/frontend-components-utilities": "^3.7.3",
     "@reduxjs/toolkit": "1.6.1",
-    "@scalprum/react-core": "^0.1.9",
```

Comment:
It looks like this change is causing test failure. RemediationButton from FEC imports @scalprum/react-core, however, the package doesn't list scalprum as a dependency in its package.json. I think we need to keep this dependency in advisor for now, but certainly this is the bug in RemediationButton component. I will open an issue.